test/unit/test-amp-story-player.js

```diff
@@ -284,4 +287,24 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       );
     });
   });
+
+  describe('Player API', () => {
+    it('calls build and layout callbacks when calling load', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      const story = win.document.createElement('a');
+      story.setAttribute('href', DEFAULT_CACHE_URL);
+      playerEl.appendChild(story);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+      const buildSpy = env.sandbox.spy(player, 'buildCallback');
+      const layoutSpy = env.sandbox.spy(player, 'layoutCallback');
+      expect(buildSpy).to.not.have.been.called;
+      expect(layoutSpy).to.not.have.been.called;
+
+      await player.load();
+
+      expect(buildSpy).to.have.been.calledOnce;
+      expect(layoutSpy).to.have.been.calledOnce;
```

Comment:
Do we have a better thing to check if the `.load()` actually worked, like checking for the story to be loaded or the iframe to be rendered? This would test the behavior more accurately, for example here if something goes wrong in the `layoutCallback` the test would still pass.